.github/workflows/release.yml

```diff
@@ -96,7 +99,7 @@ jobs:
       matrix:
         os: ['ubuntu-latest']
         variant: ['4.0']      # docs need the highest variant
-        java: ['21']          # docs need the highest java version
+        java: ['22']          # docs need the highest java version
```

Comment:
I think we should stick to the LTS versions for releases.

Reply:
Ok, I will change it.